50-main.config

```
@@ -49,7 +49,9 @@ GLIDEIN_Country = "US"
 GLIDEIN_Site = "Docker"
 GLIDEIN_ResourceName = "Docker"
 
-STARTD_ATTRS = $(STARTD_ATTRS) GLIDEIN_Country GLIDEIN_Site GLIDEIN_ResourceName WorkerGroupName IsOsgVoContainer
+GLIDEIN_ContainerTag = @CONTAINER_TAG@
```

```suggestion
GLIDEIN_ContainerTag = "@CONTAINER_TAG@"
```
Dunno if quotes are necessary here but the other attributes have them.